Makefile.PL

```diff
@@ -33,17 +33,33 @@ if ($^O eq "freebsd") {
 };
 
 sub MY::postamble {
+    my $pure_all;
     my $sharemakefile = 'share/GNUmakefile';
     if ($^O eq "freebsd") {
         # Make FreeBSD use gmake for share
-        return "GMAKE ?= \"gmake\"\n"
+        $pure_all = "GMAKE ?= \"gmake\"\n"
             . "pure_all :: $sharemakefile\n"
             . "\tcd share && \$(GMAKE) touch-po all\n";
     } else {
         # Here Linux and GNU Make is assumed
-        return "pure_all :: $sharemakefile\n"
+        $pure_all = "pure_all :: $sharemakefile\n"
             . "\tcd share && \$(MAKE) touch-po all\n";
     };
+
+    my $docker = <<'END_DOCKER';
+
+docker-build: dist
```

Comment:
I guess it should be
```
docker-build: all dist
```
Unless `make all` is run before `make docker-dist` translation will not work.